common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -155,6 +160,27 @@ public String getIssuerCacheIdentifierFromAuthority(final URL authority) {
         return authority.getHost();
     }
 
+    public String getIssuerCacheIdentifierFromTokenEndpoint() {
+        final String methodName = ":getIssuerCacheIdentifierFromTokenEndpoint";
+        URL authority = null;
+        String cacheIdentifier = null;
+
+        try {
+            authority = new URL(mTokenEndpoint);
+        } catch (MalformedURLException e) {
+            Logger.verbose(
```

Comment:
Suggest we crank this up higher than `verbose`.... `warning` or `error` feels appropriate

Reply:
Addressed